test/framework/kubernetes.go

```diff
@@ -112,6 +112,21 @@ func IsPodStatusConditionReady(pod *corev1.Pod) bool {
 	return false
 }
 
+// WaitForDeploymentRunning waits for a deployment to be running, with a specific number of pod
+func WaitForDeploymentRunning(namespace, dName string, podNb int, timeoutInMin int) error {
+	return WaitForOnOpenshift(namespace, fmt.Sprintf("Deployment %s running", dName), timeoutInMin,
```

Comment:
Will this function work on k8s as well? I ask because would be nice to also run this test cases using minikube for lightweight envs.

Reply:
yes. it should, but maybe we should rename it :)

Reply:
We can rename the WaitForOpenshift method later when we start working on the support of tests for K8s.

Reply:
But this test suppose to run on k8s. The point is to test `KogitoRuntime` CR on both envs.